src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/VisualStudioProjectSnapshotManagerAccessor.cs

```diff
@@ -2,39 +2,63 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System.ComponentModel.Composition;
-using System.Diagnostics.CodeAnalysis;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
 using Microsoft.CodeAnalysis.Razor.Workspaces;
+using Microsoft.VisualStudio.Threading;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor;
 
 [System.Composition.Shared]
 [Export(typeof(IProjectSnapshotManagerAccessor))]
 [method: ImportingConstructor]
 internal sealed class VisualStudioProjectSnapshotManagerAccessor(
-    [Import(typeof(VisualStudioWorkspace))] Workspace workspace)
+    [Import(typeof(VisualStudioWorkspace))] Workspace workspace,
+    ProjectSnapshotManagerDispatcher dispatcher,
+    JoinableTaskContext joinableTaskContext)
     : IProjectSnapshotManagerAccessor
 {
     private readonly Workspace _workspace = workspace;
+    private readonly ProjectSnapshotManagerDispatcher _dispatcher = dispatcher;
+    private readonly JoinableTaskFactory _jtf = joinableTaskContext.Factory;
+
+    private readonly object _gate = new();
```

Comment:
I'm guessing this is left over from before you were using JTF? It is unused.

Reply:
Yes, that's right. Good catch!